src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -66,12 +59,20 @@ import MaxFeeAndSubmit from '@/components/MaxFeeAndSubmit/MaxFeeAndSubmit.vue'
 // @ts-ignore
 import FormRow from '@/components/FormRow/FormRow.vue'
 
-type MosaicAttachmentType = {
-  id: MosaicId,
-  mosaicHex: string,
-  name: string,
-  amount: number,
-  uid: number,
+interface MosaicAttachment {
+  mosaicHex: string
+  /**
+   * Relative amount
+   */
+  amount: number
+}
+
+export interface MosaicAttachmentType {
+  id: MosaicId
+  mosaicHex: string
+  name: string
+  amount: number
+  uid: number
 }
```

Comment:
duplicate interfaces ... the second interface just has more fields, these can be made optional in first interface and then only use that one.

Reply:
resolved